tasks/__init__.py

```diff
@@ -992,11 +992,16 @@ def usage(ctx):
 ### Maintenance Tasks ###
 
 @task
-def set_maintenance(ctx, start=None, end=None):
+def set_maintenance(ctx, message, level=1, start=None, end=None):
```

Comment:
`message` should be optional. If it isn't provided, the behavior should be the same as before.